python/lsst/pipe/drivers/coaddDriver.py

```diff
@@ -35,13 +35,19 @@ class CoaddDriverConfig(Config):
                         doc="Run detection on the coaddition product")
     detectCoaddSources = ConfigurableField(
         target=DetectCoaddSourcesTask, doc="Detect sources on coadd")
+    fakesAdded = Field(dtype=bool, default=False, doc="Have fakes been added?")
```

Comment:
This needs a better doc, and consider a different name, see comments on related ticket